zeebe/broker-client/src/main/java/io/camunda/zeebe/broker/client/api/BrokerClusterState.java

```diff
@@ -48,4 +49,8 @@ public interface BrokerClusterState {
   String getBrokerVersion(int brokerId);
 
   PartitionHealthStatus getPartitionHealth(int brokerId, int partition);
+
+  default long getLastCompletedChangeId() {
+    return NO_LAST_COMPLETED_CHANGE_ID;
+  }
```

Comment:
:wrench: Let's avoid default implementations, it's already overwritten everywhere anyway.